src/Page/SubmitStory/View.elm

```diff
@@ -0,0 +1,124 @@
+module Page.SubmitStory.View exposing (view)
+
+import Css exposing (Style, batch, border, displayFlex, flexGrow, flexShrink, flexWrap, height, hidden, int, overflow, pct, property, rem, width, wrap)
+import Html.Styled exposing (Html, div, h2, iframe, img, text)
+import Html.Styled.Attributes as Attr exposing (css)
+import I18n.Keys exposing (Key(..))
+import I18n.Translate exposing (Language(..), translate)
+import Message exposing (Msg)
+import Page.Data
+import Shared exposing (Model)
+import Theme.Global exposing (centerContent, primaryHeader, roundedCornerStyle)
+import Theme.Markdown exposing (markdownToHtml)
+
+
+missingPageWarning : Page.Data.Page
+missingPageWarning =
```

Comment:
Not sure if this is the best pattern. Did you try to move the logic out to the Main.view like the others?

Reply:
I didn't look into putting it in `Main.view` I'll look at that now

Reply:
ah yes, Ivan's solution is much nicer